src/cable/connection.cr

```diff
@@ -102,29 +102,44 @@ module Cable
       return message(payload) if payload.command == "message"
     end
 
-    def subscribe(payload)
+    def subscribe(payload : Cable::Payload)
       channel = Cable::Channel::CHANNELS[payload.channel].new(
         connection: self,
         identifier: payload.identifier,
         params: payload.channel_params
       )
       Connection::CHANNELS[connection_identifier] ||= {} of String => Cable::Channel
       Connection::CHANNELS[connection_identifier][payload.identifier] = channel
-      # Cable.server.subscribe_channel(channel: channel, identifier: payload.identifier)
       channel.subscribed
-      Cable::Logger.info "#{payload.channel} is transmitting the subscription confirmation"
-      socket.send({type: "confirm_subscription", identifier: payload.identifier}.to_json)
+
+      if channel.subscription_rejected?
+        reject(channel)
```

Comment:
maybe an easier way would be:
```
return reject(channel) if channel.subscription_rejected?
```
so you don't make a tree of ifs?

Reply:
hummm yeah, looks like it works heheheh let me check and I push a fix